app/components/workflow-basics.js

```diff
@@ -46,7 +46,8 @@ export default Component.extend({
   didRender() {
     this._super(...arguments);
     this.send('validateDOI');
-    this.send('lookupDOI');
```

Comment:
What kind of bugs? This is required for having prepopulated DOI stuff when editing existing submissions